proto/search/search_message.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "proto/search/common.proto";
+
+option go_package = "proto-go/search;search_pb";
+
+package search;
+
+service SearchMessage {
+  rpc SearchMessage (SearchMessageRequest) returns (Message) {}
```

Comment:
You return a list of messages, so it needs to be a SearchMessageResponse object with a repeated fiels inside